client/src/components/ReviewsAddForm/index.jsx

```diff
@@ -1,11 +1,68 @@
 import React from 'react';
 import styles from './ReviewsAddForm.css';
 
-// Will need to have an onClick event listener and handler
-const ReviewsAddForm = () => (
-  <div className={styles.reviewsAddForm}>
-    <button className={styles.addButton} type="submit">Add Review</button>
-  </div>
-);
+class ReviewsAddForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      overallRating: '',
+      email: '',
+      reviewUsername: '',
+      reviewSummary: '',
+      reviewBody: '',
+      recommendProduct: '',
+    };
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ [event.target.name]: event.target.value });
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  handleSubmit(event) {
+    event.preventDefault();
+    // Need to have function that adds reviews to a tile and adds to list - function passed as props
+    // this.addReview(this.state.______);
+    // Need to reset state);
+  }
+
+  render() {
+    return (
+      <div>
+        <form onSubmit={this.handleSubmit}>
+          <h2>Write New Review</h2>
+          <h4>Overall Rating:</h4>
+          <input onChange={this.handleChange} type="text" name="overallRating" value={this.state.overallRating} placeholder="Rating: 1 - 5" />
+          <h4>What is your email?</h4>
+          <input onChange={this.handleChange} type="text" name="email" value={this.state.email} placeholder="Example: abc@123.com" />
```

Comment:
```suggestion
          <input onChange={this.handleChange} type="email" name="email" value={this.state.email} placeholder="Example: abc@123.com" />
```
for automatic validation to email address

Reply:
Made the changes. Thanks, Phil!